app/views/base/layout.scala

```diff
@@ -158,10 +158,11 @@ object layout:
     div(id := "clinput")(
       clinputLink,
       input(
-        spellcheck   := "false",
-        autocomplete := ctx.blind.toString,
-        aria.label   := trans.search.search.txt(),
-        placeholder  := trans.search.search.txt()
+        spellcheck           := "false",
+        autocomplete         := ctx.blind.toString,
+        aria.label           := trans.search.search.txt(),
+        placeholder          := trans.search.search.txt(),
+        attr("enterkeyhint") := "search"
```

Comment:
You can define custom attributes here: https://github.com/lichess-org/lila/blob/master/app/ui/scalatags.scala/#L29

Usually preferable over spelling it out everywhere since it will catch typos.

Reply:
Makes sense. No previous scala experience so I just grabbed what I could from the scalatag docs. Thank you for the tip!